libraries/rush-lib/src/api/ChangeManagement.ts

```diff
@@ -12,6 +12,8 @@ export interface IChangeFile {
 
 /**
  * Represents all of the types of change requests.
+ *
+ * todo: Replace enum
```

Comment:
As far as I can tell, this ChangeType enum only existed in memory during rush run (these numbers are never written to disk eg in change files or change logs for example), and they weren't part of the public API, so I don't think anything else would know about them either.

(I was mentioning that in case that's why you held off temporarily... I suppose since it's numeric it's also possible there's a lt/gt comparison somewhere, but I think if so it's already suspicious.  "Hotfix" and "dependency" can't be compared in a meaningful way, for example.)

Reply:
There are a bunch of instances of things like `ChangeType[ChangeType.none]`, which we'll need to handle separately. It won't change the API, so I figured I'd do that part separately.